app/views/users/show.html.erb

```diff
@@ -1,2 +1,29 @@
-<h1>This is the method show Users!</h1>
+  <section class="container margin_10px padding_10px">
+    <div class="border container_photo">
+      <img src='<%= @user.photo %>' alt='User photo' class="user_photo">
+    </div>
 
+    <div class="border user_data_container">
+      <h2 class="user_name"><%= @user.name %></h2>
+      <h2 class="post_counter">Number of posts: <%= @user.posts_counter %></h2>
+    </div>
+  </section>
+
+  <section class="border general_container margin_10px padding_10px">
+    <h3>Bio</h3>
+    <p><%= @user.bio %></p>
+  </section>
+
+  <% @user.posts.each_with_index do |post, index| %>
```

Comment:
- [ ] As per this [requirement](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/Views_project.md#:~:text=Use%20methods%20that%20you%20have%20created%20in%20the%20Blog%20App%20%2D%20processing%20data%20in%20models.), Please use the `recent_post` method you created on the user model to render the 3 most recent posts only.

Reply:
@Omar-Muhamad 
Thank you very much for your recommendations! I just applied the suggested changes.